resources/qml/ExtruderIcon.qml

```diff
@@ -48,12 +47,14 @@ Item
         UM.Label
         {
             id: extruderNumberText
-            anchors.centerIn: parent
-            text: index + 1
+            anchors.verticalCenter: parent.verticalCenter
+            anchors.left: parent.left
+            anchors.right: parent.right
+            horizontalAlignment: Text.AlignHCenter
+            text: (index + 1).toString()
             font: UM.Theme.getFont("small_emphasis")
             width: contentWidth
             height: contentHeight
```

Comment:
These should be at the top above anchors according to the style guide (This was definitely me 😅)

Reply:
https://github.com/Ultimaker/Cura/pull/14046/commits/ffb59b28942f78b7d709a545f564e7b9bfb4d75f